### PR TITLE
added psycopg2 and sqlalchemy to requirements.txt

### PR DESCRIPTION
**Pull request by vanshigupta04**
added psycopg2 and sqlalchemy to requirements.txt

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -97,4 +106,3 @@ websocket-client==1.5.0
 widgetsnbextension==4.0.5
 wordcloud==1.8.2.2
 zipp==3.12.0
-black
```
Comment (vanshigupta04) [inline requirements.txt:100]: black has been added at the top @Ayush1695

**Review by advaithsrao — COMMENTED** [review]
@Ayush1695 please review the change in the requirements file and approve

File: requirements.txt (hunk 1 above)
Comment (Ayush1695) [reply requirements.txt:100]: it wont cause issue

**Review by Ayush1695 — APPROVED** [review]

**Review by advaithsrao — APPROVED** [review]
LGTM

**Pull request merged**